src/main/frontend/components/encryption.cljs

```diff
@@ -1,176 +0,0 @@
-(ns frontend.components.encryption
```

Comment:
With encryption being removed, should we remove commented out encryption commands in commands.cljs?